tests/basic/storage_pool_test.go

```diff
@@ -9761,3 +9761,190 @@ var _ = Describe("{AddDriveMetadataPool}", func() {
+			poolList, err := GetPoolsDetailsOnNode(selectedNode)
+			log.FailOnError(err, "fialed while getting updated node pool list")
+			//first add all the new pools
+			poolLeft := maxPoolLength - len(poolList)
+			err = addNewPools(selectedNode, poolLeft)
+			drvNum, err := Inst().N.RunCommand(selectedNode, "lsblk -l -d -e 11 -n -o NAME|wc -l", node.ConnectionOpts{
+				Timeout:         defaultCommandTimeout,
+				TimeBeforeRetry: defaultCommandRetry,
+				IgnoreError:     false,
+				Sudo:            false,
+			})
+			drvNum1 := strings.TrimSpace(string(drvNum))
+			drvNum1 = strings.Trim(drvNum1, "\n")
+			drvNumInt, err := strconv.Atoi(drvNum1)
+			log.FailOnError(err, "fialed to convert to int")
+			maxDriveLimit, err := GetPoolMaxCloudDriveLimit(&selectedNode)
+			maxDriveLimitInt := int(maxDriveLimit)
+			maxDriveAllowed := maxDriveLimitInt - drvNumInt
+			err = Inst().V.RefreshDriverEndpoints()
+			log.FailOnError(err, "error refreshing volume endpoints")
+			//expand by adding disk
+			selectedNode1, err := node.GetNodeByName(selectedNode.Name)
+			log.FailOnError(err, "fialed while getting node")
+			poolList1, err := GetPoolsDetailsOnNode(selectedNode1)
+			log.FailOnError(err, "fialed while getting updated node pool list")
+
+			for maxDriveAllowed > 0 {
+				for i := 0; i < len(poolList1); i++ {
+					selectedPool, err := GetPoolWithIOsInGivenNode(selectedNode, contexts)
+					log.FailOnError(err, "error in getting pool with IO with error %s", err)
+					if selectedPool.Uuid != poolList1[i].Uuid {
+						drvSize, err := getPoolDiskSize(poolList1[i])
+						log.FailOnError(err, "error getting drive size for pool [%s]", poolList1[i].Uuid)
+						//adding 4 drives to each pool which is having one drive.so that max drive per node is accomdated
+						for j := 1; j <= 4; j++ {
+							driveSize := drvSize * uint64(j)
+							expectedSize := (poolList1[i].TotalSize / units.GiB) + driveSize
+							isjournal, err := isJournalEnabled()
+							log.FailOnError(err, "Failed to check is journal enabled")
+							expectedSizeWithJournal := expectedSize
+							if isjournal {
+								journalSize := 3 * uint64(j)
+								expectedSizeWithJournal = expectedSizeWithJournal - journalSize
+							}
+							err = Inst().V.ExpandPool(poolList1[i].Uuid, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK, expectedSize, false)
+							log.FailOnError(err, "error while expanding pool")
+							maxDriveAllowed = maxDriveAllowed - 1
+							resizeErr := waitForPoolToBeResized(expectedSize, poolList1[i].Uuid, isjournal)
+							dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Expected new size to be '%d' or '%d'", expectedSize, expectedSize-3))
+							err = Inst().V.RefreshDriverEndpoints()
+							log.FailOnError(err, "error refreshing volume endpoints")
+						}
+					}
+				}
+			}
+			err = Inst().V.RefreshDriverEndpoints()
+			log.FailOnError(err, "error refreshing volume endpoints")
+			selectedNode2, err := node.GetNodeByName(selectedNode.Name)
```

Comment:
can we pls assign proper variable instead of selectedNode2 ?

Reply:
Done